docs/operating-eck/upgrading-eck.asciidoc

```diff
@@ -99,6 +99,8 @@ Upgrading the operator results in a one-time update to existing managed resource
 
  1.6, 1.9, 2.0, 2.1, 2.2, 2.4, 2.5, 2.6, 2.7, 2.8
 
+NOTE: Stepping over one of these versions, for example upgrading ECK from 2.6 to 2.9, still triggers a rolling restart.
```

Comment:
Should it say "Skipping one of those versions"

Reply:
I am fine either way if you asked a native speaker (which I am not), it was just a gut feeling.

Reply:
But I think the most recent commit has now both (probably a git accident)

Reply:
🤦

Reply:
I'm also fine with both. Just to be sure, were you planning to go back to "Stepping" in 4c5b430?

Reply:
"Stepping" is the most recent and validated commit, yes